keyboards/viktus/minne/info.json

```diff
@@ -0,0 +1,59 @@
+{
+    "manufacturer": "Viktus Design LLC",
+    "keyboard_name": "Viktus Minne",
+    "maintainer": "BlindAssassin111",
+    "url": "https://viktus.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x5644",
+        "pid": "0x4D49"
+    },
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": true
+    },
```

Comment:
```suggestion
    },
    "build": {
        "lto": true
    },
```

Reply:
In order for LTO to be enabled in VIA, Does having it in info.JSON also enable that? Just want to check if it is needed in another location just for VIA.

Reply:
Yes, it being enabled here at the keyboard level will compile all keymaps with LTO enabled (unless they set it otherwise).

Reply:
Awesome, That makes it easy.